src/environmentd/tests/util.rs

```diff
@@ -339,6 +379,7 @@ pub struct Server {
     pub runtime: Arc<Runtime>,
     _temp_dir: Option<TempDir>,
     pub metrics_registry: MetricsRegistry,
+    pub _tracing_guard: Option<TracingGuard>,
```

Comment:
Tiniest little nit, for future reference: the leading underscore is meant to silence the unused field checker! If this is `pub` it doesn't need the leading underscore, because Rust is smart enough to realize that some downstream crate might use it. (But probably this shouldn't be `pub`?)

Reply:
Great catch! Definitely doesn't need to be `pub`, let me fix that.

It also feels like this should be a Clippy lint, so I put up https://github.com/rust-lang/rust-clippy/pull/10283, we'll see what the team thinks :)

Reply:
Since I know you're interested :) that Clippy lint finally got merged!